ansible/provision-pool-play/roles

```diff
@@ -0,0 +1 @@
+../roles
```

Comment:
we can now include these in the ansible.cfg in the playbook's parent folder.

Reply:
Can you point to an example in the community repo that can be used?